src/series/multi.js

```diff
@@ -9,7 +9,7 @@ import dataJoinUtil from '../util/dataJoin';
 // overriding where the series value is stored on the node (__series__) and
 // forcing the node datum (__data__) to be the user supplied data (via mapping).
 
-export default function() {
+export default function(className) {
```

Comment:
I did it as immutable as if the dataJoin had created the objects would be hard not to leave legacy bits. First pass was a property